fix: make merchandise list route `/`

nicolelst opened the pull request:
This commit removes routes to the old home page (pages/Home) and makes the merchandise list page(pages/MerchandiseList) the main landing page.

[SCSE-53]

nicolelst committed:
fix: make merchandise list route `/`

This commit removes routes to the old home page (pages/Home) and makes the merchandise list page(pages/MerchandiseList) the main landing page.

[SCSE-53]

nicolelst committed:
fix: make merchandise list route `/`

This commit removes routes to the old home page (pages/Home) and makes the merchandise list page(pages/MerchandiseList) the main landing page.

[SCSE-53]

nicolelst commented:
Should the drawer in the header (broken links to Home, Projects, About Us, and Merchandise) be removed or will there be sites we want the merchandise page to link to?

chanbakjsd commented:
> Should the drawer in the header (broken links to Home, Projects, About Us, and Merchandise) be removed or will there be sites we want the merchandise page to link to?

Yup. Please remove the Home/Merchandise link as well as they are broken too.

RealDyllon committed:
feat: Add Stripe Integration (#38)

* feat: add fallback component when 404 route
* feat: add typings for product & category
* refactor: change merchDetail typings to productType
* feat: skeleton for merch list
* feat: merch detail, typings, added size chart
* feat: Cart Page
* feat: checkout page
* feat: order summary page
* feat: order history page
* feat: sign up, sign in and confirm sign in to take in page component
* refactor: responsiveness on merch and change route name
* refactor: responsiveness on merch detail
* feat(cart): add cart count into header
* style(cart): fix gap between hamburger and cart
* refactor(*): Responsiveness changes, and fix max quantity to 99
* fix(cart): remove wrong equality meant for UI checking
* refactor(*): add paynow modal
* refactor(*): cart and checkout page, add payment element

Co-authored-by: Woke Orange <75624491+wokeorange@users.noreply.github.com>
Co-authored-by: Chan Wen Xu <lutherchanpublic+git@gmail.com>

YoNG-Zaii committed:
Removed Create Account & Sign Up buttons (#40)

Removed Create an Account & Sign Up buttons

YoNG-Zaii committed:
feat: Removed FooterLink (#41)

feat: Removed FooterLink

nicolelst committed:
fix: Replace references to old merchandise list route

This commit replaces any references to routes.MERCHANDISE_LIST with routes.HOME.

[SCSE-53]

nicolelst committed:
fix: Remove redundant buttons from header

This commit removes the redundant drawer with links to Home and Merchandise from the header.

[SCSE-53]

nicolelst committed:
Merge remote-tracking branch 'origin/fix/SCSE-53' into fix/SCSE-53

# Conflicts:
#	src/routes.tsx

nicolelst commented:
@chanbakjsd broken links have been fixed/removed!
